### PR TITLE
Readd alex code to ts_build_scripts

### PR DESCRIPTION
**Pull request by Greg-Boggs**

### CODE REVIEW

**Comment by wxactly** [comment]
@Greg-Boggs - Not sure about the SETTINGS_SITE and SETTINGS_LOCAL bits, but the rest of these changes look ready to merge.

File: build.sh [hunk 1]
```diff
@@ -131,8 +131,12 @@ chmod 755 $DESTINATION/sites/default
 
 # Inculde copies of the settings files that were used to build the site, for reference
 SETTINGS_SITE="$DESTINATION/profiles/$PROJECT/settings"
+
+# replace PROJECT with actual project name in local.settings.php
+sed -i '' "s/PROJECT/$PROJECT/g" $DESTINATION/profiles/$PROJECT/scripts/settings/local.settings.php
```
Comment (Greg-Boggs) [inline build.sh:136]: @mikemccaffrey please review this and decide what you think is best here. This is an older line of code that I think we either may not need now, or may want to add an if statement for.

**Comment by Greg-Boggs** [comment]
@wxactly this too.

**Comment by mikemccaffrey** [comment]
I'm not exactly sure what those lines of code are trying to do, but I'm going to say that we probably don't need them.

File: build.sh (hunk 1 above)
Comment (mikemccaffrey) [inline build.sh:136]: I'm not exactly sure what those lines of code are trying to do, but I'm going to say that we probably don't need them.

**Comment by mikemccaffrey** [comment]
This pull request seems to be removing the lines that copy the local and site.settings.php files, and then adding them back in, so I'm going to say that part is not needed.

And if we want to change all the echos to printfs, then we can just do that as a single new commit.

**Comment by Greg-Boggs** [comment]
This PR does two things. 1. It swaps echo to printf for better formatting and it adds this:

 +# uncomment RewriteBase in project's .htaccess file
+# necessary for Drupal sites running on a machine configured using ThinkShout standards
+# see https://github.com/thinkshout/ts_recipes/tree/master/brew-lamp-dev-envt
+# see https://github.com/thinkshout/ts_recipes/blob/master/environment_setup.sh
+cd $DESTINATION
+sed -i '' 's/# RewriteBase \/$/RewriteBase \//g' ./.htaccess

**Pull request closed without merging**